pmm/pmm2-new-version.groovy

```diff
@@ -0,0 +1,106 @@
+pipeline {
+    agent {
+        label 'master'
+    }
+    parameters {
+        string(
+            defaultValue: '',
+            description: 'New PMM2 version',
+            name: 'NEW_VERSION')
+    }
+    options {
+        skipStagesAfterUnstable()
+        skipDefaultCheckout()
+        disableConcurrentBuilds()
+    }
+    stages {
+        stage('Commit pmm-update') {
+            steps {
+                deleteDir()
+                withCredentials([sshUserPrivateKey(credentialsId: 'GitHub SSH Key', keyFileVariable: 'SSHKEY', passphraseVariable: '', usernameVariable: '')]) {
+                    sh '''
+                        if [ -z "${NEW_VERSION}" ]; then
+                            echo 'New PMM2 version was not set'
+                            exit 1
+                        fi
+
+                        echo "/usr/bin/ssh -i "${SSHKEY}" -o StrictHostKeyChecking=no \\\"\\\$@\\\"" > github-ssh.sh
+                        chmod 755 github-ssh.sh
+                        export GIT_SSH=$(pwd -P)/github-ssh.sh
+                        git clone git@github.com:percona/pmm-update --branch PMM-2.0
+
+                        pushd pmm-update
+                            git config --global user.email "dev-services@percona.com"
+                            git config --global user.name "PMM2 Jenkins"
+
+                            LATEST_PLAYBOOK_DIR=\$(
+                                find ansible \
+                                    -mindepth 1 \
+                                    -maxdepth 1 \
+                                    -type d \
+                                    | sort \
+                                    | tail -1
+                            )
+                            OLD_VERSION=\$(head -1 \$LATEST_PLAYBOOK_DIR/main.yml | cut -dv -f 2)
+                            MAJOR=\$(echo ${NEW_VERSION} | cut -d. -f 1)
+                            MINOR=\$(echo ${NEW_VERSION} | cut -d. -f 2)
+                            PATCH=\$(echo ${NEW_VERSION} | cut -d. -f 3 | awk -F'-' '{print $1}')
+                            BETA=\$(echo ${NEW_VERSION}  | awk -F'-' '{print $2}')
```

Comment:
Both SemVer and we call it pre-release: https://github.com/Percona-Lab/pmm-submodules/blob/d90f99bd345173ea619bafe3805ded42cacbb03e/build/bin/vars#L16